composer.json

```diff
@@ -30,6 +30,7 @@
         "ext-pdo": "*",
         "ext-session": "*",
         "ext-xml": "*",
+        "ext-net-dns2": "*",
```

Comment:
This is not correct, its not a php extension. `composer require pear/net_dns2`

Reply:
fixed